src/components/graph/TitleEditor.vue

```diff
@@ -106,7 +106,6 @@ const canvasEventHandler = (event: LiteGraphCanvasEvent) => {
     const [x, y] = group.pos
 
     const e = event.detail.originalEvent
-    // @ts-expect-error LiteGraphCanvasEvent is not typed
```

Comment:
ComfyUI-Manager supports the detection of new frontend, so we can drop this line.

https://github.com/ltdrdata/ComfyUI-Manager/commit/3701246fb16b5ecbb6f46824ce15ccf1a8664f4d